### ISSUE TITLE
Rename default template to blue_eagle, move template files into it

### ISSUE THREAD

**Issue by mahagr** [Kunena/Kunena-Forum#63]
I want to change the fact that deprecated Blue Eagle is located in our views while all other templates have all their files in template directory.  All our templates should be equal having the same structure and located in the same place.

Keeping only xml files in views/*/tmpl folder should also be useful, as it keeps the directories clean and easy to browse.

I also want to get rid of "default" as template name, mostly because of our default template tends to change from version to version.

In short this is what we should do:
- Rename template/default to template/blue_eagle
- Update KunenaTemplate and KunenaTemplateDefault20 classes defaults
- Move template PHP files from views into template/blue_eagle
